_posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md

```diff
@@ -0,0 +1,131 @@
+---
+date: 2019-07-16
+title: AWS CloudFormation Example
+categories:
+   - AWS
+description: How to enable CloudFormation and deploy a simple template.
+type: Document
+---
+
+## Background
+
+This document will show you how to enable the CloudFormation stage.  This document also assumes that you have configured an AWS account.  If you need more instructions on configuring your AWS account please refer to either [Deploying to AWS from Spinnaker (using IAM instance roles)](https://docs.armory.io/spinnaker-install-admin-guides/add-aws-account-iam/) or [Deploying to AWS from Spinnaker (using IAM credentials)](https://docs.armory.io/spinnaker-install-admin-guides/add-aws-account/) 
+
+## Preconditions
```

Comment:
Doesn't this need to run the terraform scripts to work? there is no mention of the scripts nowhere.

Reply:
The terraform scripts are for other purposes: To create IAM Roles and Policies. In this article we are assuming the user already have configured an AWS account.